### ISSUE TITLE
install via npm

### ISSUE THREAD

**Issue by thesunlover** [SebastianNette/PIXI.draggable#8]
i would like to be able to install the drag function via npm, thanks

**Comment by KosGrillis** [comment]
I would also like to do this. Is this repo still supported?

**Comment by thesunlover** [comment]
I don't think so, it has been almost an year since this request and 2 years since the last commit
